src/elements/Typography.tsx

```diff
@@ -1,23 +1,19 @@
-/**
- * Typography component that allows for easily overridding
- * color and font-weight styles.
- */
-
 import {
   TypographyProps as MuiTypogaphyProps,
   Typography as MuiTypography,
-  TypographyVariant,
-  useTheme,
 } from "@mui/material";
-import { Theme } from "@mui/material/styles";
+import { Theme, useTheme } from "@mui/material/styles";
 import { FunctionComponent } from "react";
 
 export interface TypographyProps extends MuiTypogaphyProps {
-  variant?: TypographyVariant;
   color?: keyof Theme["colors"];
   fontWeight?: "regular" | "medium" | "semi-bold" | "bold" | "extra-bold";
```

Comment:
When you select text in Figma it does not show whether it's "bold", "extra-bold", "regular", etc... but it just shows the numeric font-weight. Maybe we should consider allowing a numeric font-weight to be passed in (i.e `fontWeight={400}` instead so that our code more closely matches the designs. I find myself having to look inside this component to see which font weight to use.